app/src/main/java/org/mozilla/fenix/wallpapers/WallpapersUseCases.kt

```diff
@@ -173,24 +193,30 @@ class WallpapersUseCases(
          *
          * @param wallpaper The wallpaper to load a bitmap for.
          */
-        override suspend operator fun invoke(wallpaper: Wallpaper): Bitmap? = when (wallpaper) {
-            is Wallpaper.Local -> loadWallpaperFromDrawable(context, wallpaper)
-            is Wallpaper.Remote -> loadWallpaperFromDisk(context, wallpaper)
+        override suspend operator fun invoke(wallpaper: Wallpaper): Bitmap? = when (wallpaper.name) {
+            "amethyst", "cerulean", "sunrise" -> loadWallpaperFromDrawable(context, wallpaper)
+            "twilight-hills", "beach-vibe" -> loadWallpaperFromDisk(context, wallpaper)
             else -> null
         }
 
         private suspend fun loadWallpaperFromDrawable(
             context: Context,
-            wallpaper: Wallpaper.Local
+            wallpaper: Wallpaper
         ): Bitmap? = Result.runCatching {
+            val drawableRes = when (wallpaper.name) {
+                "amethyst" -> R.drawable.amethyst
```

Comment:
I noticed this is the third time these wallpaper names are being referenced via direct strings. Would it make sense to make them a const val or a non-translatable string?

Reply:
I would generally recommend this as well. Less chances of failure because we misspell a wallpaper name in various usage.

Reply:
Oh yeah, that's a great idea. Since I had been planning to remove them immediately afterwards I was being a little more gung-ho, but I think it's still worth doing here.